### DISCUSSION TITLE
part-ii/2021-50/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#408] [Question comments]
Part II 2021 Differential Geometry: Paper 4, Section II, F 

https://questions.tripos.org/part-ii/2021-50/

**Comment by ArthurConmy** [comment]
<details>
<summary>i)</summary>
Infinitely many, but must be disjoint.

($x^2+y^2=1$ and constant height circles)
</details> 

<details>
<summary>ii)</summary>
Infinitely many, need not be disjoint.

($x^2 + y^2 + z^2 = 1$ with a bit cut off top and bottom, and great circles through a common point)
</details> 

<details>
<summary>iii)</summary>
At most one geodesic.

(catenoid)
</details>
